plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/templates/appengine/AppEngineTemplateUtility.java

```diff
@@ -64,6 +63,22 @@ public static void createFileContent(
     }
   }
 
+  public static void copyFileContent(String outputFileLocation, String sourceName)
+      throws CoreException {
+    Preconditions.checkNotNull(outputFileLocation, "output file is null");
+    Preconditions.checkNotNull(sourceName, "source file name is null");
+
+    try (InputStream inputStream =
+        AppEngineTemplateUtility.class.getResourceAsStream("/templates/appengine/" + sourceName)) {
+      File outputFile = new File(outputFileLocation);
```

Comment:
you don't require a separate File object here. You can create the stream directly from the path, and then you could create this in the try() opener so it's autoclosed too.